src/Neo.CLI/CLI/MainService.Plugins.cs

```diff
@@ -228,24 +228,28 @@ private void OnPluginsCommand()
         {
             try
             {
-                var plugins = GetPluginListAsync().GetAwaiter().GetResult();
-                if (plugins == null) return;
-                plugins
-                .Order()
-                .ForEach(f =>
+                var plugins = GetPluginListAsync().GetAwaiter().GetResult()?.ToArray() ?? [];
+                var installedPlugins = Plugin.Plugins.ToList();
+
+                var maxLength = installedPlugins.Count == 0 ? 0 : installedPlugins.Max(s => s.Name.Length);
+                if (plugins.Length > 0)
                 {
-                    var installedPlugin = Plugin.Plugins.SingleOrDefault(pp => string.Equals(pp.Name, f, StringComparison.CurrentCultureIgnoreCase));
```

Comment:
Could of just swapped line 231 with 237. Should of been after that.

Reply:
Feel free to change it